### ISSUE TITLE
Cannot delete the last empty line

### ISSUE THREAD

**Issue by qumuq-til** [toolstack/Folio#166]
There is always a new line present in a note (seen when line numbering is on), even if there is no data in it. It's annoying and distracting. When the text ends and a user hasn't explicitly put a new line after the last character, the numbering should end at the line that has some data. The easiest way to see it to to create a new note and it will have two lines while it really should have just one.

**Comment by toolstack** [comment]
As mentioned this is intended behaviour, it might be able to be changed after #50 is implemented, but there is no timeline on that.

I'll close this for the time being as won't fix.
